addon/components/perfect-scroll/component.js

```diff
@@ -44,18 +44,57 @@ export default Ember.Component.extend({
   useKeyboard: true,
   suppressScrollX: false,
   suppressScrollY: false,
-  scrollTop: 0,
-  scrollLeft: 0,
   scrollXMarginOffset: 0,
   scrollYMarginOffset: 0,
   includePadding: false,
   theme: 'default',
 
+  _scrollTop: 0,
```

Comment:
I think it would be easier to follow if `_scrollLeft` and `_scrollTop` defined after `theme` together.